lib/extensions/ar_adapter/ar_pglogical.rb

```diff
@@ -0,0 +1,228 @@
+module PgLogical
+  def pglogical_installed?
+    exec_query("SELECT EXISTS(SELECT * FROM pg_available_extensions WHERE name = 'pglogical')").cast_values.first
+  end
+
+  # Enables pglogical postgres extensions
+  def pglogical_enable
+    enable_extension("pglogical")
+    enable_extension("pglogical_origin")
+  end
+
+  # Node Management
+  #
+
+  # Creates a node
+  #
+  # @param name [String]
+  # @param dsn [String] external connection string to the node
+  def pglogical_node_create(name, dsn)
+    raw_connection.exec_params("SELECT pglogical.create_node($1::VARCHAR, $2::VARCHAR)", [name, dsn])
```

Comment:
@matthewd Running that line gives me this failure:
```
Failure/Error: select_value("SELECT pglogical.create_node($1, $2)", "Replication", [name, dsn])

NoMethodError:
  undefined method `value_for_database' for "test-node":String
# /home/ncarboni/.gem/ruby/2.2.4/gems/activerecord-5.0.0.beta2/lib/active_record/connection_adapters/postgresql_adapter.rb:590:in `block in exec_no_cache'
# /home/ncarboni/.gem/ruby/2.2.4/gems/activerecord-5.0.0.beta2/lib/active_record/connection_adapters/postgresql_adapter.rb:590:in `map'
# /home/ncarboni/.gem/ruby/2.2.4/gems/activerecord-5.0.0.beta2/lib/active_record/connection_adapters/postgresql_adapter.rb:590:in `exec_no_cache'
# /home/ncarboni/.gem/ruby/2.2.4/gems/activerecord-5.0.0.beta2/lib/active_record/connection_adapters/postgresql_adapter.rb:580:in `execute_and_clear'
# /home/ncarboni/.gem/ruby/2.2.4/gems/activerecord-5.0.0.beta2/lib/active_record/connection_adapters/postgresql/database_statements.rb:13:in `select_value'
# ./lib/extensions/ar_adapter/ar_pglogical.rb:20:in `pglogical_node_create'
# ./spec/lib/extensions/ar_pglogical_spec.rb:44:in `block (4 levels) in <top (required)>'
```
I also get the same thing if I use the casts in SQL. Is there a different type I'm supposed to be using for the bind parameters over ruby's `String`?